.github/workflows/rdev-update-for-pr.yml

```diff
@@ -24,14 +24,6 @@ permissions:
   contents: write
 
 jobs:
```

Comment:
hm i think we should still run the push tests when you update the PR, but i agree we don't need the push tests to succeed to start building images.

regardless removing this job definition entirely i'm pretty sure will break since the `rdev-tests` at the bottom here requires `push-tests`. i think if we just remove the requirement of needing `push-tests` on `build_images` and keep the job definition + requirement on `rdev-tests` that would work well?

Reply:
> regardless removing this job definition entirely i'm pretty sure will break since the rdev-tests at the bottom here requires push-tests. 

This would break the workflow if we don't remove the requirement from rdev-tests.

> i think if we just remove the requirement of needing push-tests on build_images and keep the job definition + requirement on rdev-tests that would work well?

That's wont fix the problem of workflows canceling randomly.

Reply:
Thanks for catching the fact that `push-tests` got left in as needed for `rdev-tests`. I've removed it now.